TouchPortalAPI/client.py

```diff
@@ -22,7 +22,8 @@
 from pyee import ExecutorEventEmitter
 from concurrent.futures import Executor, ThreadPoolExecutor
 from threading import Event, Lock
-from .tools import Tools
+from .tools import , Log
```

Comment:
I messed this up during rebase: it should be `from .tools import Tools, Log`. This is fixed in the next commit on this branch, but this commit is not atomic w/out a minor fix.